### PR TITLE
:wrench: fix: example apps urls

### PR DESCRIPTION
**Pull request by codingben**

### CODE REVIEW

**Review by bakayolo — CHANGES_REQUESTED** [review]
Good job.

However, `Repository URL` should only contains the git URL.
For the path, you can add an extra step and provide the `Subfolder Path`.

File: docs/examples/angular/README.md [hunk 1]
```diff
@@ -24,7 +25,7 @@ This will take a few minutes to deploy. Then click on the **Access** tab and cop
 - Clone the repository to a folder of choice.
 
 ```
-$ git clone https://github.com/kintohub-examples/angular
+$ git clone https://github.com/kintoproj/kinto-examples
 
 $ cd angular
```
Comment (sumitkolhe) [inline docs/examples/angular/README.md]: Action - `cd kinto-examples/angular`

**Review by sumitkolhe — CHANGES_REQUESTED** [review]

**Review by sumitkolhe — APPROVED** [review]
LGTM

**Review by bakayolo — APPROVED** [review]
LGTM

**Pull request merged**